src/psyclone/psyGen.py

```diff
@@ -5678,6 +5834,33 @@ def rhs(self):
 
         return self._children[1]
 
+    @staticmethod
+    def create(lhs, rhs):
+        '''Create an Assignment instance given lhs and rhs child instances.
+
+        :param lhs: the PSyIR node containing the left hand side of \
+            the assignment.
+        :type lhs: :py:class:`psyclone.psyGen.Node`
+        :param rhs: the PSyIR node containing the right hand side of \
+            the assignment.
+        :type rhs: :py:class:`psyclone.psyGen.Node`
+
+        :returns: an Assignment instance.
+        :rtype: :py:class:`psyclone.psyGen.Assignment`
+
```

Comment:
Docstring the errors (also in all remaining `create` methods)

Reply:
Done.